cas-server-support-oauth/src/main/java/org/jasig/cas/support/oauth/CentralOAuthService.java

```diff
@@ -216,4 +219,20 @@ AuthorizationCode grantAuthorizationCode(TokenType type, String clientId, String
      * @throws InvalidScopeException the invalid scope exception
      */
     Map<String, Scope> getScopes(Set<String> scopeSet) throws InvalidScopeException;
+
+    /**
+     * Get a list of all the refresh tokens for a given client of the id specified.
+     *
+     * @param clientId the client id
+     * @return a list of refresh tokens
+     */
+    Collection<RefreshToken> getClientRefreshTokens(String clientId);
```

Comment:
`CentralOauthService` now exposes the refresh token list of a specified client to `OAuth20RevokeClientTokensController`. Same for the access token list below.